tesseract_kinematics/include/tesseract_kinematics/core/utils.h

```diff
@@ -301,27 +302,30 @@ inline bool isWithinLimits(const Eigen::Ref<const Eigen::Matrix<FloatType, Eigen
  * @param limits The joint limits of the robot
  */
 template <typename FloatType>
-inline std::vector<FloatType> getRedundantSolutions(const FloatType* sol, const Eigen::MatrixX2d& limits)
+inline std::vector<Eigen::Matrix<FloatType, Eigen::Dynamic, 1>>
```

Comment:
It might be nice to have an alias for something like VectorXFloatType here.

Reply:
Done.